src/Microsoft.TestPlatform.Extensions.TrxLogger/XML/XmlPersistence.cs

```diff
@@ -858,26 +847,27 @@ private string ProcessXPathQuery(string queryIn)
             // fix the empty namespaces to a temp prefix, so xpath query can understand them
             string[] parts = queryIn.Split(new char[] { '/' }, StringSplitOptions.None);
 
-            string query = string.Empty;
+            StringBuilder query = new StringBuilder();
```

Comment:
Doubt this is any faster than just string.

Reply:
Maybe it is not faster, but in my opinion it good pattern to use StringBuilder in all cases when we concatenate the strings.

Reply:
responded above.